### PR TITLE
Issue #DP-000 chore: Bulk job request api onboarding

### PR DESCRIPTION
**Pull request by venkateshwarans**

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5193,3 +5193,75 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+- name: programBulkJobRequestCreate
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5197]: syntax issues are present in this yaml @venkateshwarans please check all the below apis as well, they have same issues.

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5193,3 +5193,75 @@ kong_apis:
...
+    request_path: "{{ program_service_prefix }}/v1/process/create"
+    upstream_url: "{{ program_service_url }}/program/v1/process/create"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'appAccess'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5207]: invalid Access control list @venkateshwarans please check all the below apis as well, they have same issues.

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -5193,3 +5193,75 @@ kong_apis:
...
+    request_path: "{{ program_service_prefix }}/v1/process/search"
+    upstream_url: "{{ program_service_url }}/program/v1/process/search"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5261]: this is an admin functionality and can't be contentCreate

Comment (venkateshwarans) [reply ansible/roles/kong-api/defaults/main.yml:5261]: this is user doing search for the bulk process he initiated.

File: ansible/roles/kong-api/defaults/main.yml [hunk 4]
```diff
@@ -5193,3 +5193,75 @@ kong_apis:
...
+    request_path: "{{ program_service_prefix }}/v1/process/update"
+    upstream_url: "{{ program_service_url }}/program/v1/process/update"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5243]: is this an admin functionality or any user can perform this update?

File: ansible/roles/kong-api/defaults/main.yml (hunk 3 above)
Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml]: yes, this should have admin role as he can enumerate the jobs

File: ansible/roles/kong-api/defaults/main.yml (hunk 4 above)
Comment (venkateshwarans) [reply ansible/roles/kong-api/defaults/main.yml:5243]: Any user can perform this.

Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml:5243]: looks good @venkateshwarans

**Pull request merged**